src/docs/sphinx/t_conduit_docs_tutorial_python_basics_out.txt

```diff
@@ -9,115 +9,165 @@ Updating test list for fixtures
-111: test_003_basics_object_and_list (t_conduit_docs_tutorial_python_basics.Conduit_Tutorial_Python_Basics) ... ok
-111: test_004_basics_object_and_list_itr (t_conduit_docs_tutorial_python_basics.Conduit_Tutorial_Python_Basics) ... ok
-111: test_005_basics_mem_spaces (t_conduit_docs_tutorial_python_basics.Conduit_Tutorial_Python_Basics) ... ok
-111: test_006_basics_bw_style (t_conduit_docs_tutorial_python_basics.Conduit_Tutorial_Python_Basics) ... ok
-111: test_007_basics_bw_style_from_native (t_conduit_docs_tutorial_python_basics.Conduit_Tutorial_Python_Basics) ... ok
-111: 
-111: ----------------------------------------------------------------------
-111: Ran 7 tests in 0.003s
-111: 
-111: OK
-111: BEGIN_EXAMPLE("py_basics_very_basic")
-111: 
-111: my: "data"
-111: 
-111: END_EXAMPLE("py_basics_very_basic")
-111: BEGIN_EXAMPLE("py_basics_hierarchial")
-111: 
-111: my: "data"
-111: a: 
-111:   b: 
-111:     c: "d"
-111:     e: 64.0
-111: 
-111: total bytes: 15
-111: 
-111: END_EXAMPLE("py_basics_hierarchial")
-111: BEGIN_EXAMPLE("py_basics_object_and_list")
-111: 
-111: object_example: 
-111:   val1: "data"
-111:   val2: 10
-111:   val3: 3.1415
-111: list_example: 
-111:   - 0
-111:   - 1
-111:   - 2
-111:   - 3
-111:   - 4
-111: 
-111: END_EXAMPLE("py_basics_object_and_list")
-111: BEGIN_EXAMPLE("py_basics_object_and_list_itr")
-111: 
-111: object_example: 
-111:   val1: "data"
-111:   val2: 10
-111:   val3: 3.1415
-111: list_example: 
-111:   - 0
-111:   - 1
-111:   - 2
-111:   - 3
-111:   - 4
-111: 
-111: val1: "data"
-111: val2: 10
-111: val3: 3.1415
-111: 
-111: 0
-111: 1
-111: 2
-111: 3
-111: 4
-111: END_EXAMPLE("py_basics_object_and_list_itr")
-111: BEGIN_EXAMPLE("py_basics_mem_spaces")
-111: 
-111: mem_spaces: 
-111:   0x7ff883407c70: 
-111:     path: "my"
-111:     type: "allocated"
-111:     bytes: 5
-111:   0x7ff883417250: 
-111:     path: "a/b/c"
-111:     type: "allocated"
-111:     bytes: 2
-111:   0x7ff88340fc80: 
-111:     path: "a/b/e"
-111:     type: "allocated"
-111:     bytes: 8
-111: total_bytes_allocated: 15
-111: total_bytes_mmaped: 0
-111: total_bytes_compact: 15
-111: total_strided_bytes: 15
-111: 
-111: END_EXAMPLE("py_basics_mem_spaces")
-111: BEGIN_EXAMPLE("py_basics_bw_style")
-111: 
-111: test: 100
-111: 
-111: END_EXAMPLE("py_basics_bw_style")
-111: BEGIN_EXAMPLE("py_basics_bw_style_from_native")
-111: 
-111: {
-111:   "test": {"dtype":"int64","number_of_elements": 1,"offset": 0,"stride": 8,"element_bytes": 8,"endianness": "little"}
-111: }
-111: END_EXAMPLE("py_basics_bw_style_from_native")
-1/1 Test #111: t_conduit_docs_tutorial_python_basics ...   Passed    0.46 sec
+113: Test command: /Users/harrison37/Work/github/llnl/conduit/uberenv_libs/spack/opt/spack/darwin-mojave-x86_64/clang-10.0.0-apple/python-3.7.6-zd24bom2hns4o42jlzdptijz54ogai7c/bin/python3.7 "-B" "-m" "unittest" "-v" "t_conduit_docs_tutorial_python_basics"
```

Comment:
This is also pretty hard to validate; all of these lines got updated because the prefix was updated (i.e. `111` to `113`).